lib.Makefile

```diff
@@ -338,6 +338,31 @@ define update_replace_pin
 		fi'
 endef
 
+# Get the latest release tag from projectcalico/go-build.
+GO_BUILD_REPO=https://github.com/projectcalico/go-build.git
+define get_go_build_version
```

Comment:
i think this should give the option for a specific version based on go version.

while on master it is fine to get the latest, as we create new branches, they should go based on their go version?